components/expression_language/syntax.rst

```diff
@@ -127,18 +127,55 @@ Working with Functions
 ----------------------
 
 You can also use registered functions in the expression by using the same
-syntax as PHP and JavaScript. The ExpressionLanguage component comes with one
-function by default: ``constant()``, which will return the value of the PHP
-constant::
+syntax as PHP and JavaScript. The ExpressionLanguage component comes with the
+following functions by default:
+
+* ``constant()``
+* ``enum()``
+
+``constant()`` function
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This function will return the value of a PHP constant::
 
     define('DB_USER', 'root');
 
     var_dump($expressionLanguage->evaluate(
         'constant("DB_USER")'
     ));
 
+    // This also works with class constants
```

Comment:
Let's make this a plain text followed by a new code block and add a namespace for the class.

Maybe let's not reuse DB_USER